templates/Error/error403.html.twig

```diff
@@ -3,5 +3,9 @@
 {% block body %}
     <h1>{{ 'error.messages.unauthorized'|trans }}</h1>
     <p>{{ 'error.messages.unauthorizedMessage'|trans }}</p>
-    <a href="{{ app.request.headers.get('referer') }}" class="btn btn-outline-primary">{{ 'go back'|trans }}</a>
-{% endblock %}
\ No newline at end of file
+    {% if app.request.headers.get('referer') != "" %}
+        <a href="{{ app.request.headers.get('referer') }}" class="btn btn-outline-primary">{{ 'go back'|trans }}</a>
+    {% else %}
+        <a href="{{ fallbacks.get('redirect_path') }}" class="btn btn-outline-primary">{{ 'go back'|trans }}</a>
```

Comment:
What is the default value of this fallback?

Reply:
This will be "/" but this get's setup in  the application skeleton

Reply:
@jonasdekeukelaere tested and changed to the |default method 0216df8a840b01ce1bf6b30413c8bd2f3f29bfdc

Reply:
> This will be "/" but this get's setup in the application skeleton
Dus het is sowieso niet backwards compatibel? Unless you do something (untested) like `fallbacks.get('redirect_path', '/')`

Reply:
@tijsverkoyen for older projects you could still add the redirect path to parameters.fallbacks array in services.yaml The application skeleton just adds these values by default

Reply:
yeah. That is not backwards compatible. Unless `/` is the default fallback.

Reply:
@tijsverkoyen 3bb6a9f474482c7ee25de5a3562cddb7b956353f
`fallbacks.get('redirect_path', '/')` works so `/` will be the fallback